### PR TITLE
CI: add vectore test

### PR DESCRIPTION
**Pull request by xiaokamikami**
Add vectore's test for CI

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ jobs:
       - name: Setup env
         run: |
           echo "NEMU_HOME=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+          git checkout master
```
Comment (poemonsense) [inline .github/workflows/ci.yml:21]: no need for this. CI will automatically checkout to the HEAD commit to run tests

Comment (xiaokamikami) [reply .github/workflows/ci.yml]: ok

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -128,3 +129,28 @@ jobs:
...
+    name: nemu - V-extension
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup env
+        run: |
+          echo "NEMU_HOME=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+          echo "TEST_HOME=/nfs/home/share/ci-workloads/V-extension-tests" >> $GITHUB_ENV
+          git fetch
+          git checkout rv64v
```
Comment (poemonsense) [inline .github/workflows/ci.yml:143]: To test rv64v branch only, should use `branches: [ rv64v ]` to specify the condition of CI and this CI file is updated only on the rv64v branch. No need to merge into master.

What's the target of this PR? To test RVV on rv64v branch?

If so, you can modify this PR over the rv64v branch, and set the CI condition. Then, github will automatically run CI on the rv64v branch.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -128,3 +128,28 @@ jobs:
...
+      - name: Build NEMU with V extension as difftest ref
+        run: |
+          make riscv64-rvv-diff-spike_defconfig
+          make -j
+          make clean-all
+      - name: Run Vector-hmmer with Spike DiffTest
+        run: |
+          make riscv64-rvv-diff-spike_defconfig
+          echo "#define CONFIG_DETERMINISTIC 1" >> ./include/generated/autoconf.h
```
Comment (poemonsense) [inline .github/workflows/ci.yml:152]: is this `CONFIG_DETERMINISTIC ` required?

File: .github/workflows/ci.yml (hunk 2 above)
Comment (xiaokamikami) [reply .github/workflows/ci.yml:143]: Yes, we only want to do this test on rv64v for now, we will need to add it to master later

**Pull request merged**